homeassistant/components/gios/sensor.py

```diff
@@ -59,7 +59,6 @@ class GiosSensorEntityDescription(SensorEntityDescription):
     GiosSensorEntityDescription(
         key=ATTR_AQI,
         name="AQI",
-        device_class=SensorDeviceClass.AQI,
         value=None,
     ),
     GiosSensorEntityDescription(
```

Comment:
Looks like some unrelated changes snuck in

Reply:
Yah, hang off on reviewing this. I'm going to force push once other changes are merged and clean this up. I just wanted to start capturing changes.